### PR TITLE
nix: Remove profileEnv

### PR DESCRIPTION
**Pull request by akshaymankar**
The overridden NIX_PATH doesn't seem to be used anywhere.

### CODE REVIEW

**Comment by lucendio** [comment]
> The overridden NIX_PATH doesn't seem to be used anywhere.

[I think it is - again](https://github.com/wireapp/wire-server-deploy/pull/443).

**Comment by arianvp** [comment]
`NIX_PATH` is just there as a weird convenience such that `nix-shell -p <package here>` works inside the direnv'd directory. It's not _needed_ it was  a convenience hack that Florian added.

**Comment by akshaymankar** [comment]
I am a little unsure on what convenience it provides. Is it just that if you want to run`nix-shell -p <something>`, it will use the nixpkgs pinned in the repo?

**Comment by arianvp** [comment]
That's the only convenience yes.  I'm not sure if it's worth it. 

Especially when  `nix 3.0` comes (any time now!) with the Flake support; `NIX_PATH` won't even really exist anymore...

**Comment by akshaymankar** [comment]
Cool, I will fix conflicts and ask for reviews again soon :)

**Comment by akshaymankar** [comment]
@arianvp I fixed the conflicts, please feel free to review it and merge if you approve.

File: default.nix [hunk 1]
```diff
@@ -6,21 +6,9 @@ let
...
-  profileEnv = pkgs.writeTextFile {
-    name = "profile-env";
-    destination = "/.profile";
-    # This gets sourced by direnv. Set NIX_PATH, so `nix-shell` uses the same nixpkgs as here.
-    text = ''
-      export NIX_PATH=nixpkgs=${toString pkgs.path}
-    '';
-  };
-
-  pythonForAnsible = pkgs.python3.withPackages (p: [ p.boto p.boto3 p.six p.cryptography ]);
```
Comment (flokli) [inline default.nix:18]: Is this unused?

**Comment by flokli** [comment]
The convenience here also ensures we all use the same terraform minor version, in case we need to `nix-shell` one of these in.

Is this so much of a burden to not keep around?

**Comment by akshaymankar** [comment]
Stale. Closing, will reopen if I get bothered by this.

**Pull request closed without merging**